$OpenBSD: patch-configure_lua,v 1.1 2017/03/01 20:42:57 naddy Exp $

Accept more compilers than a hardcoded gcc.

(SetDriversClang() is also a no-op) 

--- configure.lua.orig	Sun Nov 13 16:41:35 2016
+++ configure.lua	Tue Feb 28 19:34:03 2017
@@ -362,10 +362,9 @@ function OptCCompiler(name, default_driver, default_c,
 		else
 			if ExecuteSilent("cl") == 0 then
 				option.driver = "cl"
-			elseif ExecuteSilent("g++ -v") == 0 then
-				option.driver = "gcc"
 			else
-				error("no c/c++ compiler found")
+				-- Assume GCC compatible
+				option.driver = "${CC}"
 			end
 		end
 		--setup_compiler(option.value)
@@ -374,12 +373,9 @@ function OptCCompiler(name, default_driver, default_c,
 	local apply = function(option, settings)
 		if option.driver == "cl" then
 			SetDriversCL(settings)
-		elseif option.driver == "gcc" then
-			SetDriversGCC(settings)
-		elseif option.driver == "clang" then
-			SetDriversClang(settings)
 		else
-			error(option.driver.." is not a known c/c++ compile driver")
+			-- Assume GCC compatible
+			SetDriversGCC(settings)
 		end
 
 		if option.c_compiler then settings.cc.c_compiler = option.c_compiler end
